pkg/internal/golang/go.go

```diff
@@ -0,0 +1,108 @@
+package golang
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/giantswarm/e2e-harness/pkg/internal/docker"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	dockerImage = "quay.io/giantswarm/golang:1.11.1"
```

Comment:
That should be configurable via flags for the commands of interest. Then we do not have to rebuild the project to update the version.

Reply:
It's fine to rebuild the projects. Setting flag would be a pain. We'd have to update all circle configs instead of rebuilding project once.